tests/tests/RDMResponderTest.cpp

```diff
@@ -138,11 +174,41 @@ class RDMResponderTest : public testing::Test {
     RDMResponder_Initialize(&settings);
   }
 
+  void SetupSensors(ResponderDefinition *responder_def) {
+    responder_def->sensors = SENSOR_DEFINITIONS;
+    responder_def->sensor_count = 1;
+    g_responder->sensors = g_sensors;
+
+    for (unsigned int i = 0; i < NUMBER_OF_SENSORS; i++) {
+      g_sensors[i].present_value = 0u;
+      g_sensors[i].lowest_value = 0u;
+      g_sensors[i].highest_value = 0u;
+      g_sensors[i].recorded_value = 0u;
+      g_sensors[i].should_nack = false;
+    }
+  }
+
  protected:
   UID m_controller_uid;
   UID m_our_uid;
   StrictMock<MockPIDHandler> m_pid_handler;
 
+  unique_ptr<RDMRequest> BuildGetRequest(uint16_t pid,
```

Comment:
We have similar ones in the ModelTest, but this is for the RDMResponder test, which isn't a model.